CalculateDifference.java

```diff
@@ -0,0 +1,23 @@
+import java.util.HashMap;
+import java.util.Map;
+
+public class CalculateDifference {
+
```

Comment:
@msharma105 The code is correct. But can you move the logic to a function and pass the two arrays as parameters to that function? Avoid writing any logic inside the main() function from now on.
Always create a new function and call that function from the main() function or other function.